bench/hist/Main.hs

```diff
@@ -178,37 +178,37 @@ main = shakeArgs shakeOptions {shakeChange = ChangeModtimeAndDigest} $ do
 
   priority 0 $
     [ build -/- "*/*.csv",
-      build -/- "*/*.benchmark-gcStats"
+      build -/- "*/*.benchmark-gcStats",
+      build -/- "*/*.log"
     ]
-      &%> \[outcsv, _outGc] -> do
+      &%> \[outcsv, _outGc, outLog] -> do
         let [_, _, exp] = splitDirectories outcsv
         samples <- readSamples
         liftIO $ createDirectoryIfMissing True $ dropFileName outcsv
         let ghcide = dropFileName outcsv </> "ghcide"
             ghcpath = dropFileName outcsv </> "ghc.path"
         need [ghcide, ghcpath]
         ghcPath <- readFile' ghcpath
-        verb <- getVerbosity
         withResource ghcideBenchResource 1 $ do
           Stdout res <-
             command
-              [ EchoStdout True,
+              [ EchoStdout False,
```

Comment:
Have you seen `FileStdout` https://hackage.haskell.org/package/shake-0.19.1/docs/Development-Shake.html#v:FileStdout? Could that be used to avoid reading it as a buffer, then writing it to a file?

Reply:
Oh, that's useful, thanks!

Reply:
I really like `cmd_` and friends btw, they are super ergonomic.

Reply:
Thanks! I think I've spent about as long on cmd_ as I have on all of the rest of Shake :)